src/app/learn/get-started/basics/modules/index.html

```diff
@@ -22,7 +22,7 @@
         In the command prompt, run the following command from your SPA's root directory, where <stache-code>[package-name]</stache-code> is the name of the package that contains the module.
       </p>
       <sky-code-block>
-        npm install --save --save-exact [package-name]
+        npm install [package-name] --save --save-exact
```

Comment:
I actually just swapped the order in this command in the past week based on @Blackbaud-BobbyEarl's preference. I changed it here and in one other location, and I'm planning to make a similar change to all the instances in the Components section in an upcoming PR. Did you get feedback from someone else that we should keep `--save --save exact` at the end?

Reply:
Oops, nope. I just switched it to match the way it was listed for the components, so I can edit it back to what it was.